code/datums/helper_datums/butchering.dm

```diff
@@ -346,6 +348,9 @@ var/global/list/animal_butchering_products = list(
 	/mob/living/carbon/monkey							= list(/datum/butchering_product/skin/monkey, TEETH_FEW),
 	/mob/living/simple_animal/rabbit					= list(/datum/butchering_product/rabbit_ears, /datum/butchering_product/rabbit_foot),
 	/mob/living/simple_animal/snail						= list(/datum/butchering_product/snail_carapace),
+	/mob/living/simple_animal/hostile/wolf				= list(/datum/butchering_product/skin/wolf, TEETH_LOTS),
+	/mob/living/simple_animal/hostile/wolf/pliable		= list(/datum/butchering_product/skin/wolf, TEETH_LOTS),
+	/mob/living/simple_animal/hostile/wolf/alpha		= list(/datum/butchering_product/skin/wolf, TEETH_LOTS),
```

Comment:
you don't need to include this 3 separate times, it gets inherited from wolf

the reason bear does it for every type is they have different pelt types

Reply:
i attempted to have it work on one wolf to failure, for some reason it didnt inherit

Reply:
It doesn't inherit. It uses species_type

Instead of how you've done it here, have just the base wolf, but go into the wolf file (`code/modules/mob/living/simple_animal/hostile/wolf.dm`) and add this:
`species_type = /mob/living/simple_animal/hostile/wolf`
to just the base wolf, it will inherit that